src/legacy/core_plugins/timelion/server/handlers/lib/parse_sheet.js

```diff
@@ -33,7 +33,8 @@ export default function parseSheet(sheet) {
       if (e.expected) {
         throw new Error(
           i18n.translate('timelion.serverSideErrors.sheetParseErrorMessage', {
-            defaultMessage: 'Expected: {expectedDescription} @ character {column}',
+            defaultMessage: 'Expected: {expectedDescription} at character {column}',
+            description: 'This would be for example: "Expected: a quote at character 5',
```

Comment:
Closing quote is missed
```suggestion
            description: 'This would be for example: "Expected: a quote at character 5"',
```

_Note_: In `en.json` it will look like `"comment": "This would be for example: \"Expected: a quote at character 5\""`. If we revert quotes, we will get rid of backslashes: `description: "This would be for example: 'Expected: a quote at character 5'",`